src/app/login/login.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="login-container">
+    <div class="login-text-box">
+        <h3>Login</h3>
+        <p style="font-size: 80%;">Get access to your Orders, Wishlist and Recommendations</p>
+    </div>
+    <form class="login-form">
+        <div class="wrap-input100">
+            <input class="input100" type="text" name="email">
```

Comment:
If the field will receive email, why the type doesn't reflect that? Is input type="email" good and helpful on mobile?